Jenkinsfile

```diff
@@ -127,7 +127,18 @@ pipeline {
                     sh "docker build --cache-from ${env.BUILD_IMAGE} --label cachebust=${env.CACHEBUST} --target build --build-arg CACHEBUST=${env.CACHEBUST} -t ${env.BUILD_IMAGE} ."
                     // Create a running container (https://stackoverflow.com/a/38308399).
                     sh "docker create --name=builder -it ${env.BUILD_IMAGE}"
-                    sh "docker start builder"
+                    sh "docker start builder && \
+                        i=0; \
+                        while [ \$(docker inspect --format '{{.State.Status}}' builder) != running ]; do \
+                            docker ps; \
+                            if [ \$i -gt 7 ]; then \
```

Comment:
Nit, If the intention here is to wait for a minute(60sec) would better use `-gt 6`.

Reply:
No, of course the intention was to wait for 70s. :grin: 

Will fix together with the commit message typo.

Reply:
`-gt 6` is still not correct, has to be `-ge 6`. It doesn't matter, though, which is also why I didn't bother thinking about this earlier.